bot/voice.go

```diff
@@ -0,0 +1,99 @@
+package bot
+
+import (
+	"log"
+	"strings"
+	"time"
+
+	"github.com/bwmarrin/discordgo"
+)
+
+var activeReceiver *AudioReceiver
+
+func stopActiveReceiver() {
+	if activeReceiver != nil {
+		activeReceiver.Stop()
+		activeReceiver = nil
+	}
+}
+
+func dcCmd(ctx CommandCtx) error {
+	if ctx.Message.Author.ID == morthisId {
+		var uId string
+
+		if len(ctx.Args) < 1 {
+			uId = venovaId
+		} else {
+			uId = strings.Trim(ctx.Args[0], "<@>")
+		}
+
+		if uId == venovaId {
+			stopActiveReceiver()
+		}
+		err := disconnectUserFromVC(ctx.Session, ctx.Message.GuildID, uId)
+		if err != nil {
+			return ctx.Reply("Error disconnecting from voice channel.")
+		}
+		if uId == venovaId {
+			return ctx.Reply("Disconnected Venova from the server.")
+		} else {
+			return ctx.Reply("Disconnected user from voice channel.")
+		}
+	}
+	return ctx.Reply("stinky")
+}
+
+func disconnectUserFromVC(session *discordgo.Session, guildID, userID string) error {
```

Comment:
Isn't this function just wrapping `GuildMemberMove`? Why not just move the code to where it's being called since we're only using it in once place, and the only added behavior is some logging.